File: app/src/main/java/com/nagot/lastfm/ui/albuminfo/AlbumInfoFragmentMvpView.java
package com.nagot.lastfm.ui.albuminfo;

import com.nagot.lastfm.base.BaseMvpView;
import com.nagot.lastfm.model.ArtistInfo;

/**
 * Created by IanNagot on 16/03/2018.
 */

public interface AlbumInfoFragmentMvpView extends BaseMvpView {

    void updateData(ArtistInfo artistInfo);
}

File: app/src/main/java/com/nagot/lastfm/ui/albuminfo/AlbumInfoFragmentMvpView.kt
package com.nagot.lastfm.ui.albuminfo

import com.nagot.lastfm.base.BaseMvpView
import com.nagot.lastfm.model.AlbumInfo

/**
 * Created by Nagot on 18/03/2018.
 */
interface AlbumInfoFragmentMvpView: BaseMvpView {

    fun updateData(albumInfo: AlbumInfo)
}